### ISSUE TITLE
Difference between "openseg.pytorch" and "HRNet-Semantic-Segmentation"

### ISSUE THREAD

**Issue by iamimage** [openseg-group/openseg.pytorch#31]
Dear Authors,

Could you clarify the difference between the "HRNet-OCR" at  (1. https://github.com/openseg-group/openseg.pytorch)
and the "HRNet-OCR" at (2. https://github.com/HRNet/HRNet-Semantic-Segmentation/tree/HRNet-OCR)?

- Is the difference very minor and can be ignored? 
- Or the difference is just about the pytorch version (e.g., version 0.41 for 1. and version 1.1 for 2. in the training using the Cityscapes dataset)?
- What's your advice (which one to start with) for the beginners?

Many thanks!

**Comment by PkuRainBow** [comment]
Thanks for your interest in our work. There exist some minor differences that can be ignored.

The official Sync-BN within pytorch 1.1 or 1.2 has some bugs and caused degraded performance.

I would like to try the openseg.pytorch if you use pytorch0.4.1 and HRNet-OCR or MMSegmentation code base if you use pytorch1.5/1.6~

**Comment by ritaiask** [comment]
I noticed one module :
dsn_head(in openseg)                   = 3x3Conv + bnrelu + dropout + 1x1conv,
while aux_head ( in HRNET-OCR) = 1x1Conv + bnrelu +                  1x1 conv

**Comment by PkuRainBow** [comment]
@ritaiask Good points. Both choices work and their performances are close.  Applying 1x1 conv is mainly for better efficiency.

**Comment by ydhongHIT** [comment]
> Thanks for your interest in our work. There exist some minor differences that can be ignored.
> 
> The official Sync-BN within pytorch 1.1 or 1.2 has some bugs and caused degraded performance.
> 
> I would like to try the openseg.pytorch if you use pytorch0.4.1 and HRNet-OCR or MMSegmentation code base if you use pytorch1.5/1.6~

pytorch 1.3 with HRNet-OCR is ok?

**Comment by hsfzxjy** [comment]
> > Thanks for your interest in our work. There exist some minor differences that can be ignored.
> > The official Sync-BN within pytorch 1.1 or 1.2 has some bugs and caused degraded performance.
> > I would like to try the openseg.pytorch if you use pytorch0.4.1 and HRNet-OCR or MMSegmentation code base if you use pytorch1.5/1.6~
> 
> pytorch 1.3 with HRNet-OCR is ok?

It's not guaranteed since we don't know whether PyTorch 1.3 has introduced any breaking changes.
